routes/api/stories.js

```diff
@@ -20,7 +20,7 @@ router.get('/stories',asyncHandler(async (req,res,next) => {
 }));
 
 
-router.get('/stories/following',restoreUser, asyncHandler(async(req,res,next) => {
+router.get('/following',restoreUser, asyncHandler(async(req,res,next) => {
```

Comment:
Use `requireAuthApi` instead of `restoreUser`. This will ensure that a user exists before trying to use `res.locals.user.id` in the queries.